plutus-tx-plugin/test/IntegerLiterals/NoStrict/9.6/integerLiterals-NoStrict.pir.golden

```diff
@@ -0,0 +1,76 @@
+(program
+  1.1.0
+  (let
+    (nonrec)
+    (termbind
+      (strict)
+      (vardecl integerNegate (fun (con integer) (con integer)))
+      (lam x (con integer) [ [ (builtin subtractInteger) (con integer 0) ] x ])
+    )
+    (lam
+      x
+      (con integer)
+      (let
+        (nonrec)
+        (termbind
+          (strict)
+          (vardecl y (con integer))
+          [ integerNegate (con integer 24680135792468013579) ]
```

Comment:
This is pretty surprising! So in 9.2 GHC turns a double negation into nothing, but in 9.6 it turns it into two calls to `integerNegate`, even with `NegativeLiterals` turned on??? Is `NegativeLiterals` broken?

Reply:
No, GHC doesn't do anything. It is my first implementation of handling `integerNegate` that cancels double negation!